cmd/ak/cmd/deployments/create.go

```diff
@@ -14,7 +14,7 @@ import (
 var activate bool
 
 var createCmd = common.StandardCommand(&cobra.Command{
-	Use:     "create <--build-id=...> <--env=...> [--activate]",
+	Use:     "create <--build-id=...> <--project=...> [--activate]",
```

Comment:
Same question as in `cmd/deploy.go`: if build objects already have a project ID field, why does the user need to specify it manually here?

Reply:
see above